src/core/Util.js

```diff
@@ -18,16 +18,6 @@ export function extend(dest, ...args) {
 	return dest;
 }
 
-// @function create(proto: Object, properties?: Object): Object
-// Compatibility polyfill for [Object.create](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object/create)
-export const create = Object.create || (function () {

```

Comment:
So apparently this was one of those IE8 compatibility polyfills.

It's safe to depend on `Object.create`, but I anticipate some plugins to still depend on `L.Util.create`, and fail. Still, this had to be done.

Reply:
Yeah, now is the time where we can start shedding some of that weight. It's important to advocate using the standard Web APIs.